apps/server/src/modules/tool/controller/tool.controller.ts

```diff
@@ -27,4 +41,15 @@ export class ToolController {
 		const mapped: Lti11LaunchResponse = this.lti11ResponseMapper.mapAuthorizationToResponse(authorization);
 		return mapped;
 	}
+
+	@Post('tools')
```

Comment:
You could add more docs e.g. for possible responses: 
https://docs.nestjs.com/openapi/operations

Reply:
I added more tags.